solidity-v1/dashboard/src/pages/OverviewPage.jsx

```diff
@@ -121,51 +114,40 @@ const OverviewPage = (props) => {
 
 const OverviewFirstSection = () => {
   return (
-    <Tile
-      title="Make the most of your KEEP tokens by staking them and earning rewards with the token dashboard."
-      titleClassName="h2 mb-2"
-    >
-      <div className="start-staking">
-        <div className="start-staking__btn">
-          <Link
-            to={DelegationPage.route.path}
-            className="btn btn-primary btn-lg"
-          >
-            start staking
-          </Link>
-        </div>
-        <div>
-          <h4 className={"text-grey-40"}>
-            Get KEEP tokens on the following exchanges:
-          </h4>
-          <Icons.BalancerLogo />
-          &nbsp;
-          <a
-            target="_blank"
-            rel="noopener noreferrer"
-            href={
-              "https://balancer.exchange/#/swap/ether/0x85eee30c52b0b379b046fb0f85f4f3dc3009afec"
-            }
-            className="text-black mr-2"
-          >
-            Balancer
-          </a>
-          &nbsp;
-          <Icons.UniswapLogo style={{ verticalAlign: "text-top" }} />
-          &nbsp;
-          <a
-            target="_blank"
-            rel="noopener noreferrer"
-            href={
-              "https://app.uniswap.org/#/swap?inputCurrency=ETH&outputCurrency=0x85eee30c52b0b379b046fb0f85f4f3dc3009afec"
-            }
-            className="text-black"
-          >
-            Uniswap
-          </a>
+    <section className="upgrade-threshold-banner tile flex row center">
+      <Icons.KeepTUpgrade className="threshold-upgrade-page__explanation__icon" />
+      <div>
+        <h4 className={"h2 mb-2 text-grey-70"}>
+          Upgrade your KEEP to T and get started with the Threshold Network.
+        </h4>
+        <div className="upgrade-threshold-banner__content">
+          <div className="upgrade-threshold-banner__upgrade-btn">
+            <Link
+              to={ThresholdUpgradePage.route.path}
+              className="btn btn-primary btn-lg"
+            >
+              upgrade now
+            </Link>
```

Comment:
Shouldn't we use `NavLink` component here?

Reply:
You are right! And actually I found some other places where we also sohuld use `NavLink` instead of just normal `Link`. I fixed all of this here - 7192d4d2cd420bc39f7c057ff6e387ffcfa00da8